tests/integration/update.bats

```diff
@@ -895,22 +895,23 @@ EOF
 @test "update per-device iops/bps values" {
 	[ $EUID -ne 0 ] && requires rootless_cgroup
 
-	# We need a major number of any disk device. Usually those are partitioned,
-	# with the device itself having minor of 0, and partitions are 1, 2...
-	major=$(awk '$2 == 0 {print $1; exit}' /proc/partitions)
-	if [ "$major" = "0" ] || [ "$major" = "" ]; then
-		echo "=== /proc/partitions ==="
-		cat /proc/partitions
-		echo "==="
-		skip "can't get device major number from /proc/partitions (got $major)"
+	# Need major:minor for any block device (but not a partition).
+	dev=$(lsblk -dno MAJ:MIN | head -1 | tr -d ' \t\n')
+	if [ -z "$dev" ]; then
+		echo "=== lsblk -d ===" >&2
+		lsblk -d >&2
+		echo "===" >&2
+		skip "can't get device from lsblk"
```

Comment:
Let's not skip this test, what do you think?

Reply:
Agreed, assuming that currently it works.

If it currently works, it's probably better to make it a hard failure, so that we can decide whether to skip or fail when we run into failures (which may be a more specific "skip" for expected conditions).